src/main/java/soolutions/hometaskapp/core/Amount.java

```diff
@@ -0,0 +1,40 @@
+package soolutions.hometaskapp.core;
+
+// TODO(vivek): Write docs for each public class
+public final class Amount {
+  private final double value;
+  private final String currency;
+
+// TODO(vivek): Check if there is an exisiting library for currency handling
```

Comment:
Do we need an external library?

Reply:
Not always, if the library is internal then there shouldn't be any issue.